charts/monitoring-logging/values.yaml

```
@@ -99,7 +99,7 @@ grafana:
       disable_signout_menu: false
       allow_sign_up: true
     server:
-      domain: "https://logs.test.nl/"
+      domain: "logs.test.nl/"
```

a domain does not have a trailing slash, so
```suggestion
      domain: "logs.test.nl"
```